Polyhedron/demo/Polyhedron/Scene_polyhedron_item.cpp

```diff
@@ -117,15 +117,15 @@ struct Scene_polyhedron_item_priv{
   }
 
 
-  void compute_normals_and_vertices(const bool colors_only = false) const;
+  void compute_normals_and_vertices(const bool colors_only = false, const bool is_two_side = false) const;
```

Comment:
I do not really understand why the parameter here is named `is_two_side`. At the very least, there should be an 's': `is_two_sides`.

Anyway, there is already a good name: `draw_two_sides`.